test/ILLink.RoslynAnalyzer.Tests/UnconditionalSuppressMessageCodeFixTests.cs

```diff
@@ -0,0 +1,512 @@
+    int M2() => M1();
+}
+class D
+{
+    [UnconditionalSuppressMessage(""Trimming"", ""IL2026:Methods annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code"", Justification = ""<Pending>"")]
+    public int M3(C c) => c.M1();
+    public class E
+    {
+        [UnconditionalSuppressMessage(""Trimming"", ""IL2026:Methods annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code"", Justification = ""<Pending>"")]
+        public int M4(C c) => c.M1();
+    }
+}
+public class E
+{
+    public class F
+    {
+        [UnconditionalSuppressMessage(""Trimming"", ""IL2026:Methods annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code"", Justification = ""<Pending>"")]
+        public int M5(C c) => c.M1();
+    }
+}
+";
+			await VerifyUnconditionalSuppressMessageCodeFixWithRUC (
+				test,
+				fixtest,
+				baselineExpected: new[] {
+				// /0/Test0.cs(7,17): warning IL2026: Using method 'C.M1()' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. message.
+				VerifyCSUSMwithRUC.Diagnostic (RequiresUnreferencedCodeAnalyzer.DiagnosticId).WithSpan (7, 17, 7, 21).WithArguments ("C.M1()", "message", ""),
+				// /0/Test0.cs(11,27): warning IL2026: Using method 'C.M1()' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. message.
+				VerifyCSUSMwithRUC.Diagnostic (RequiresUnreferencedCodeAnalyzer.DiagnosticId).WithSpan (11, 27, 11, 33).WithArguments ("C.M1()", "message", ""),
+				// /0/Test0.cs(14,31): warning IL2026: Using method 'C.M1()' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. message.
+				VerifyCSUSMwithRUC.Diagnostic (RequiresUnreferencedCodeAnalyzer.DiagnosticId).WithSpan (14, 31, 14, 37).WithArguments ("C.M1()", "message", ""),
+				// /0/Test0.cs(21,31): warning IL2026: Using method 'C.M1()' which has 'RequiresUnreferencedCodeAttribute' can break functionality when trimming application code. message.
+				VerifyCSUSMwithRUC.Diagnostic (RequiresUnreferencedCodeAnalyzer.DiagnosticId).WithSpan (21, 31, 21, 37).WithArguments ("C.M1()", "message", "")
+				},
+				fixedExpected: Array.Empty<DiagnosticResult> ());
+		}
+
+		[Fact]
+		public async Task RequiresAssemblyFilesDiagnosticFix ()
+		{
+			var test = @"
+using System.Diagnostics.CodeAnalysis;
+public class C
+{
+    [RequiresAssemblyFiles(Message = ""message"")]
+    public int M1() => 0;
+    int M2() => M1();
+}
+class D
+{
+    public int M3(C c) => c.M1();
+    public class E
+    {
+        public int M4(C c) => c.M1();
+    }
+}
+public class E
+{
+    public class F
+    {
+        public int M5(C c) => c.M1();
```

Comment:
Same here, why do I care that the original class `D` is internal but `E` is `public`?